ambari-server/src/main/python/ambari_server/dbConfiguration_linux.py

```diff
@@ -598,6 +597,8 @@ class PGConfig(LinuxDBMSConfig):
   )
 
   def __init__(self, options, properties, storage_type):
+    if PGConfig.PG_INITDB_CMD is None:
+      raise FatalException(1, "Cannot find postgresql-setup script.")
```

Comment:
Move `FatalException` here in order to raise this exception only when `__init__` called with `createPGConfig()`.